lib/client.c

```diff
@@ -677,7 +679,8 @@ SF_CONNECT *STDCALL snowflake_init() {
         sf->token = NULL;
         sf->master_token = NULL;
         sf->login_timeout = SF_LOGIN_TIMEOUT;
-        sf->network_timeout = SF_NETWORK_TIMEOUT;
+        sf->network_timeout = 0;
```

Comment:
Why change the network timeout?

Reply:
roll back the change made in previous PR to keep the behavior of existing connection parameters.